$OpenBSD: patch-include_ruby_ruby_h,v 1.1.1.1 2018/01/15 23:57:42 jeremy Exp $

Fix GCC error: #pragma GCC diagnostic not allowed inside functions
when compiling some extensions.

Index: include/ruby/ruby.h
--- include/ruby/ruby.h.orig
+++ include/ruby/ruby.h
@@ -2227,7 +2227,7 @@ ERRORFUNC(("variable argument length doesn't match"), 
 # define rb_scan_args_verify_count(fmt, varc) \
     ((varc)/(rb_scan_args_count(fmt, varc)))
 
-# ifdef __GNUC__
+# if 0
 # define rb_scan_args_verify(fmt, varc) \
     ({ \
 	int verify; \
